src/CADL.Extension/Emitter.Csharp/src/type/InputEnumTypeValue.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export interface InputEnumTypeValue {
+    Name: string;
+    Value: any;
```

Comment:
Can we restrict the `Value` to only string or number type (per [cadl spec](https://github.com/microsoft/cadl/blob/main/docs/tutorial.md#enums))?

Reply:
Yes, enum value can be string and number. when we finalize the actually type we will support, I will update the definition, could be string|number.